_posts/2022-10-18-trino-summit-2022-teaser-2.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Trino Summit 2022: Federating humans and data"
+author: "Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/summit-logo.png
+---
+
+Trino has long been the de facto standard to querying large data sets over your
+cloud or on-prem storage, also known as data lakes. This Trino Summit's theme 
+instead will showcase Trino's other claim to fame: query federation. In essence,
+Trino is a query engine providing an access point that exposes ANSI SQL across 
+[multiple data sources]({{site.url}}/docs/current/connector/).
+
+I can't help but bring up the analogy of how Trino federates heterogeneous data
+while this Trino Summit will federate many of us in the community form all
+corners of the earth. It really brings an appreciation to the international
+reach this technology has and makes me look forward to future in-person events!
+
+I urge you to join us either in-person or virtually if you are a fan of Trino,
+big data, open source, data engineering, Java, or all the above! This conference
+is free and takes place on November 10th. Check out the details below!
+
+<!--more-->
+
+## Register for the summit
+
+Trino Summit is being held at the Commonwealth Club in San Francisco, CA.
+Make sure you register quickly for in-person registration, as it is limited to
+250 seats! Virtual registration is also picking up quickly and is limited to
+1500 registrants. Register today before spots run out!
+
+<div class="card-deck spacer-30">
+    <a class="btn btn-pink" href="https://www.starburst.io/info/trinosummit/">
+        Register now
+    </a>
+</div>
+<div class="spacer-30"></div>
+
+### Trino Summit 2022 teaser
+
+Check out the teaser for this year's Trino SUmmit and get ready to Federate 'em
+all!
+
+<iframe src="https://www.youtube.com/embed/o2MJvRKG14M" width="800" height="500" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px;
+margin-bottom:5px; max-width: 100%;" allowfullscreen="">
+</iframe>
+
+
+## Announcing the second round of sessions and speakers
+
+As we announced in the [previous summit teaser]({% post_url
+2022-09-22-trino-summit-2022-teaser %}), we have already announced some of our
+exciting lineup of speakers! The topics range from architectures like data mesh
+and data lakehouse, to running Trino at scale with fault-tolerant execution, and
+of course, query federation. 
+
+We have a full roster planned but here is a glance at a few full confirmed
```

Comment:
where is "here" .. 

also "fully confirmed"

We have a full schedule planned for the day. Following are details on another batch of confirmed sessions.